syntax = "proto3";
import "google/protobuf/any.proto";
package core;

service ServerClient {
  rpc Serving (ServingRequest) returns (ServingReply) {}
}

message ServingRequest {
  string service_name = 1;
  google.protobuf.Any payload = 2;
}

message ServingReply {
  google.protobuf.Any payload = 3;
}